convert-stage-4.sh

```diff
@@ -130,6 +132,9 @@ install_files() {
   sudo install -d -m 755 ${data_dir}/${dataconfdir}
   sudo install -d -m 755 ${data_dir}/${databootdir}
 
+  install -d -m 755 ${primary_dir}/${sysconfdir}/scripts/
```

Comment:
`sudo` on the install dir also :sweat_smile:

Reply:
Doh!